netbox/dcim/models.py

```diff
@@ -1510,9 +1570,15 @@ class Device(ChangeLoggedModel, ConfigContextModel, CustomFieldModel):
 
     class Meta:
         ordering = ['name']
-        unique_together = [
-            ['rack', 'position', 'face'],
-            ['virtual_chassis', 'vc_position'],
+        constraints = [
+            models.UniqueConstraint(
+                fields=['rack', 'position', 'face'],
+                name='device_rack_position_face_key'
+            ),
+            models.UniqueConstraint(
+                fields=['virtual_chassis', 'vc_position'],
+                name='device_virtualchassis_vcpos_key'
```

Comment:
Be sure to use the full exact field name; no need to abbreviate it.

Reply:
Will do. Sorry about that, that’s a remnant from the initial PR. There are a few places like this one, I’ll patch them up!

Reply:
I removed the abbreviation, but stayed true with my original non-hyphenated version for now. I’ll fix these once we’ve decided on a version to use!